packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -102,16 +113,68 @@ export interface MetricExporter {
 /**
  * Base interface for aggregators. Aggregators are responsible for holding
  * aggregated values and taking a snapshot of these values upon export.
+ *
+ * Use {@link Aggregator} instead of this BaseAggregator.
  */
-export interface Aggregator {
+interface BaseAggregator {
```

Comment:
Why do we need a new `BaseAggregator` instead of adding a kind to `Aggregator` ?